service/src/test/java/bio/terra/tanagra/service/accesscontrol/AouWorkbenchAccessControlTest.java

```diff
@@ -270,4 +254,28 @@ void annotationKey() {
     assertServiceListWithReadPermission(
         USER_4, ResourceType.ANNOTATION_KEY, cohort2Id, true, annotationKey2Id);
   }
+
+  // Specific cases for AoU
+  private void assertUnderlayPermissions(UserId user, ResourceId resource) {
+    assertTrue(impl.isAuthorized(user, Permissions.empty(resource.getType()), resource));
+    assertServiceListWithReadPermission(USER_1, ResourceType.UNDERLAY, null, false);
+  }
+
+  private void assertStudyPermissions(UserId user, ResourceId resource, Action... actions) {
+    assertTrue(impl.isAuthorized(user, Permissions.empty(resource.getType()), resource));
+    assertStudyNoListPermissions(user, resource.getType(), actions);
```

Comment:
Similar to underlays above, is it possible to reuse an existing method here instead? e.g. `assertServiceListWithReadPermission(USER_1, ResourceType.STUDY, /*parent*/ null, /*isAllResources*/ false)`

Reply:
Cant use because - Listing for studies is controlled on AoU RW side. so except for list actions other actions are allowed depending on user's workspace access level

Reply:
I see, OK.